cluster-provision/centos8/scripts/dnsmasq.sh

```diff
@@ -40,4 +48,4 @@ ip6tables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
 ip6tables -A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 ip6tables -A FORWARD -i br0 -o eth0 -j ACCEPT
 
-exec dnsmasq --interface=br0 --enable-ra -d ${DHCP_HOSTS} --dhcp-range=192.168.66.10,192.168.66.200,infinite --dhcp-range=::10,::200,constructor:br0,static
+exec dnsmasq --interface=${INTERFACES} --enable-ra -d ${DHCP_HOSTS} ${IPV4_RANGES} --dhcp-range=::10,::200,constructor:br0,static
```

Comment:
With this change, we may need to force removal of secondary network profiles on nodes (https://github.com/nmstate/kubernetes-nmstate/blob/main/cluster/up.sh#L14). Otherwise, nodes may start up inconsistently with different default network each time they boot.

Reply:
> > I haven't used the secondary network in VMIs. I only link virt-handlers to that secondary network for migration purposes.
> 
> ok, but I am worried about the existing network tests that may assume no dhcp/ra server exists on these.
> I want to make sure that if we connect a secondary interface to the vmi, it will not by default enable that interface with dynamic IP. I can check it on Sunday, if you are ok waiting until then.

@ormergi has checked this one (Or, please correct me here).
If this the server is set for secondary networks, with a NAD that just defines a bridge CNI with no IPAM, the VMI guests we use will automatically get an IP & routes because this is NetworkManager default.

If this is indeed correct, we have a problem (even though the current tests somehow pass).
Just because without doing any configuration on the guests (usually through cloud-init-network), we will end up with addresses deployed there in secondary interfaces.

An alternative could be to dedicate a specific secondary network for this migration network and not enable the dnsmasq on all. WDYT?